src/main/java/gov/usgs/wma/waterdata/TransformDao.java

```diff
@@ -26,17 +26,17 @@
 	@Value("classpath:sql/getDiscreteGroundWater.sql")
 	protected Resource selectQuery;
 
-	public List<DiscreteGroundWater> getDiscreteGroundWater(String fieldVisitIdentifier) {
+	public List<DiscreteGroundWater> getDiscreteGroundWater(String locationIdentifier) {
 		List<DiscreteGroundWater> rtn = Collections.emptyList();
 		try {
 			String sql = new String(FileCopyUtils.copyToByteArray(selectQuery.getInputStream()));
 			rtn = jdbcTemplate.query(
 					sql,
 					new DiscreteGroundWaterRowMapper(),
-					fieldVisitIdentifier
+					locationIdentifier
 					);
 		} catch (EmptyResultDataAccessException e) {
-			LOG.info("Couldn't find {} - {} ", fieldVisitIdentifier, e.getLocalizedMessage());
+			LOG.info("Couldn't find {} - {} ", locationIdentifier, e.getLocalizedMessage());
```

Comment:
I don't think this happens (and should log an error if it did and now swallow the error).